tests/go/go-kafka-serde/client.go

```diff
@@ -52,6 +56,7 @@ type TestClient struct {
 	producer   *kafka.Producer
 	consumer   *kafka.Consumer
 	count      int
+	timeout    int
```

Comment:
Can we make this a time.Duration so people in other places don't need to understand the units